tox.ini

```diff
@@ -1,9 +1,15 @@
 [tox]
 requires =
     tox>=4
-envlist = py3, lint, type
+envlist = py3, lint
 isolated_build = True
 
+[testenv]
+deps =
+    .[dev]
+commands =
+    python3 -m unittest
```

Comment:
Why `unittest` instead of `pytest`?

Reply:
Because it's built-in and less dependencies is always better.

Reply:
While I generally agree that unnecessary dependencies should be avoided, the `if __name__ == "__main__":` block in `tests/test_cli.py` is clunky. Will `python3 -m unittest` result in discovery and execution of the tests in `tests/test_cli.py` if you remove that block?

Reply:
Yes, this is just so that if you can run the tests as a python script if you want. I removed it.